src/main/java/ladder/domain/Names.java

```diff
@@ -0,0 +1,43 @@
+package ladder.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Names {
+    public static final int MIN_PERSON_COUNT = 2;
+    public static final int NOT_FOUND = -1;
+    private List<Name> names;
+
+    public Names(String[] requestName) {
+        this(Arrays.stream(requestName).map(name -> new Name(name)).collect(Collectors.toList()));
```

Comment:
stream의 가독성을 위해서 `stream` 이후의 `.`마다 개행을 해주시면 좋을 것 같습니다.
```suggestion
        this(Arrays.stream(requestName)
                           .map(Name::new)
                           .collect(Collectors.toList()));
```

Reply:
네~